imports/plugins/core/layout/client/templates/layout/header/header.html

```diff
@@ -2,7 +2,7 @@
 <div class="container-fluid">
   <div class="nav" role="navigation">
 
-    <div class="row">
+    <div class="row" >
```

Comment:
@bolorundurowb: it there any reason for the whitespace after `class="row"` and before the closing angle bracket.